content/number-theory/chapter.tex

```diff
@@ -48,3 +48,14 @@ \section{Estimates}
 	$\sum_{d|n} d = O(n \log \log n)$.
 
 	The number of divisors of $n$ is at most around 100 for $n < 5e4$, 500 for $n < 1e7$, 2000 for $n < 1e10$, 200\,000 for $n < 1e19$.
+
+\section{Mobius Function}
+\[
+	\mu(n) = \begin{cases} 0 & n \textrm{ is not square free}\\ 1 & n \textrm{ has even number of prime factors}\\ -1 & n \textrm{ has odd number of prime factors}\\\end{cases}
+\]
+  Mobius Inversion:
+  \[ g(n) = \sum_{d|n} f(d) \Leftrightarrow f(n) = \sum_{d|n} \mu(d)g(n/d) \]
+
+  Other useful formulas/forms:
+  \[ \sum_{d | n} \mu(d) = [ n = 1] \]
+  \[ g(n) = \sum_{n|d} f(d) \Leftrightarrow f(n) = \sum_{n|d} g(d)\mu(d/n)\]
```

Comment:
`g(n) = \sum_{n|d} f(d)` seems to be a sum over infinitely many d?

Reply:
https://math.stackexchange.com/questions/1965342/does-the-mobius-inversion-theorem-hold-when-sum-function-is-over-multiples-inste

and

https://codeforces.com/blog/entry/59962?#comment-475400

It's possible if `f(d) = 0` for some high enough d.

Reply:
Oh, I see. I wonder if it's worth a comment, but not obvious what would fit. "(if convergent)" maybe